arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/StpCall.java

```diff
@@ -88,16 +96,15 @@ private void stpCallForCalls(Hub hub) {
           default -> throw new IllegalArgumentException(
               "STP module triggered for a non CALL-type instruction");
         };
-    this.warm = frame.isAddressWarm(to);
+    this.warm = frame.isAddressWarm(to) || isPrecompile(to);
```

Comment:
the only non-formatting change

Reply:
That is a potentially a bug. Did you notice this with the `ToyExecutionEnvironmentV2` or with replay tests @letypequividelespoubelles ?

Reply:
with replayTest, but the bug comes from a @FlorianHuc 's block with a BESU node